_posts/2018-02-08-figure.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title:  Quality figures from OMERO.figure
+categories: tools
+---
+
+*This is a repost from the Figure.openmicroscopy.org blog where its creator
+Will Moore talks about how OMERO.figure works:*
+
+I recently read an excellent guide by Benjamin Names on
+[How to Create Publication-Quality Figures](http://cellbio.emory.edu/bnanes/figures/).
+
+It describes the goals of scientific figure creation (accuracy, quality, transparency)
+and a very thorough workflow to achieve these goals. The key is to understand your
+data and how it is stored, manipulated and displayed. In particular, it is important
+to minimise the number of steps where data is transformed and perform lossy steps
+as late as possible in the figure creation process.
+
+Benjamin documents specific tools that he uses for his workflow such as ImageJ for images and
+Inkscape for figure layout. But much of his workflow can also be applied to
+OMERO.figure since it was designed with the same principals in mind. 
+
+I highly recommend you read the guide above, as it provides a lot of background
+information on how computers handle vector and raster data.
+The steps of Benjamin's guide that can be replicated in OMERO.figure are
+described below.
+
+
+## Preparing figure components (High-bit-depth images)
+
+The OMERO server stores your original microscope files and can
+render them as 8-bit images using your chosen rendering settings.
+Single-color LUTs can be applied to each channel over a specified
+intensity range and channels can be toggled on and off. None of these
+changes will alter the original microscope data.
+OMERO.figure does not require you to save 8-bit images before
+creating a figure, since all rendering is done 'live' within
+the figure app itself after importing images, as described below.
+
+
+## Figure layout
+
+OMERO.figure is similar to Inkscape and Adobe Illustrator in that it
```

Comment:
need for copyright or trademark symbol?

Reply:
We didn't have it originally

Reply:
Looked it up and there's no requirement to do so and it has no legal significance anyway

Reply:
thanks